libpspmath/PSPBUILD

```diff
@@ -0,0 +1,32 @@
+pkgname=libpspmath
+pkgver=r4.eb44107
+pkgrel=1
+pkgdesc="Import of mrmrice's libpspmath vfpu library for psp homebrew"
+arch=('mips')
+url="https://github.com/tufty/libpspmath"
+license=('BSD-3-Clause')
+groups=('pspdev-default')
+depends=()
+makedepends=()
+optdepends=()
+source=("git+https://github.com/tufty/libpspmath.git#commit=eb44107538fa84bab299b5d7dcf71df64a674da3")
+sha256sums=('SKIP')
+
+pkgver() {
+    cd "$pkgname"
+    # Set the revision of the repo as version
+    printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+}
+
+build() {
+    cd "$pkgname"
+    make --quiet $MAKEFLAGS || { exit 1; }
+}
+
+package() {
+    cd "$pkgname"
+    make --quiet $MAKEFLAGS install
```

Comment:
This does not copy the files to the package. We should probably do this instead:
```suggestion
        mkdir -m 775 -p "$pkgdir/psp/sdk/lib"
        install -m 644 libpspmath.a "$pkgdir/psp/sdk/lib/"
        mkdir -m 775 -p "$pkgdir/psp/sdk/include"
	install -m 644 pspmath.h "$pkgdir/psp/sdk/include/"
```